admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,52 @@
 How to Upgrade Your ownCloud Server
 ===================================
 
-There are three ways to upgrade your ownCloud server:
+We recommend that you keep your ownCloud server upgraded regularly installing all point releases and major releases, without skipping any of them, as doing so increases the risk of errors. 
+
+- Major releases are 8.0, 8.1, 8.2, 9.0, 9.1, and 10.0. 
+- Point releases are intermediate releases for each major release, e.g., 8.0.9, 8.1.3, and 10.0.2. 
+
+When an update is available for your ownCloud server, you will see a notification at the top of your ownCloud Web interface. 
+When you click the notification, it will bring you here.
+
+Before beginning an upgrade, please keep the following points in mind:
+
+- Review :doc:`the release notes </release_notes>` for important information about the needed migration steps during that upgrade to help ensure a smooth upgrade process.
+- Skipping major releases is not supported. However *you can* migrate from 8.2 or 9.0 straight to 10.0.   
```

Comment:
@kdslkdsaldsal @pako81 I was sure I didn't change anything that wasn't already there (somewhere). However, I'll update the docs shortly.